spec/kokoro-run-all.sh

```diff
@@ -173,16 +173,15 @@ if [[ $RUN_ALL_TESTS = "1" ]]; then
 else
   SPEC_DIRS=$(find $CHANGED_DIRS -type d -name 'spec' -path "*/*" -not -path "*vendor/*" -exec dirname {} \; | sort | uniq)
   echo "Running tests in modified directories: $SPEC_DIRS"
-  for PRODUCT in $SPEC_DIRS
-  do
+  for PRODUCT in $SPEC_DIRS; do
     # Run Tests
     echo "[$PRODUCT]"
     export TEST_DIR="$PRODUCT"
     pushd "$REPO_DIRECTORY/$PRODUCT/"
 
     start_time="$(date -u +%s)"
 
-    (bundle update && bundle exec rspec --format documentation) || set_failed_status
+    (bundle update && bundle exec rspec --format RspecJunitFormatter --out sponge_log.xml) || set_failed_status
```

Comment:
Might want to have both formatters so we get nice stdout for humans and the XML file for computers. Tested this locally and it works. Also, if you `tee` to a log file with the same base name, Sponge/Build Status will know what to do with it (once they're uploaded as artifacts).

```suggestion
    (bundle update && bundle exec rspec --format documentation --format RspecJunitFormatter --out sponge_log.xml | tee sponge_log.log) || set_failed_status
```